views/signup.ejs

```diff
@@ -0,0 +1,53 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/open-fonts@1.1.1/fonts/inter.min.css">
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css">
```

Comment:
i wonder if we need to adjust the margin for some of the div, input and button...

Reply:
maybe in the future but for right now the site doesn't have to look perfect

Reply:
Will resolve in this issue: https://github.com/garageScript/databases/issues/110